fpiweb/templates/fpiweb/index.html

```diff
@@ -38,32 +38,43 @@ <h2>Inventory Management</h2>
     <div class="row">
         <a class="btn btn-primary" href="
             {% url 'fpiweb:manual_checkout_box' %}">
-            Checkout (Consume Product in) a Box
+            Checkout (Consume Product In) a Box
         </a>
     </div>
 
-    <br/>
+     <br/>
+
+    <div class="row">
+        <a class="btn btn-primary" href="
+        {% url 'fpiweb:manual_move_box' %}">Move a box
```

Comment:
Should "Move a Box" be capitalized?

Reply:
Yes it should. Thanks for catching this.